root.spec

```diff
@@ -118,12 +133,14 @@ TARGET_PLATF=
 
 cat <<\EOF >> MyConfig.mk
 CFLAGS+=-D__ROOFIT_NOBANNER
+CFLAGS+=${USER_CXXFLAGS}
 CXXFLAGS+=-D__ROOFIT_NOBANNER
+CXXFLAGS+=${USER_CXXFLAGS}
 EOF
```

Comment:
Note, that MyConfig.mk will go way with a switch to ROOT 6.04.XX, which is built with CMake.